src/components/Cards.js

```diff
@@ -7,10 +7,24 @@ export default function Cards({ data }) {
     <Card>
       <Div>
         <Title>{data.date}</Title>
-        <span>{data.time}</span>
+        <div>
+          <span>{data.time}</span>-<span>{data.timeto}</span>
+        </div>
       </Div>
       <span>{data.water}</span>
       <span>{data.target}</span>
+      <Invisible>
```

Comment:
Is this data invisible the whole time or is it toggled and shown sometimes? Invisible sounds to me a little like it's never visible. Maybe furtherInformation or anything like that would make it clearer if data is shown from time to time 😉

Reply:
It is for a function in the future. I want to toggle some data in the cards